stl/inc/numeric

```diff
@@ -670,7 +682,22 @@ _NODISCARD constexpr common_type_t<_Mt, _Nt> lcm(const _Mt _Mx, const _Nt _Nx) n
         return 0;
     }
 
-    return static_cast<_Common>((_Mx_magnitude / _STD gcd(_Mx_magnitude, _Nx_magnitude)) * _Nx_magnitude);
+#ifndef _DEBUG
+    if (!_STD _Is_constant_evaluated()) {
+        return static_cast<_Common>((_Mx_magnitude / _STD gcd(_Mx_magnitude, _Nx_magnitude)) * _Nx_magnitude);
+    } else
+#endif // ^^^ !defined(_DEBUG) ^^^
+    {

```

Comment:
Suggestion: This is a plain `if`, not `if constexpr`, so we could early `return` to simplify the visual control flow. There's an early return above too.